src/views/tables/PositionsTable/PositionsMarginCell.tsx

```diff
@@ -0,0 +1,54 @@
+import styled, { AnyStyledComponent } from 'styled-components';
+
+import { type SubaccountPosition } from '@/constants/abacus';
+import { ButtonShape } from '@/constants/buttons';
+import { USD_DECIMALS } from '@/constants/numbers';
+
+import { IconName } from '@/components/Icon';
+import { IconButton } from '@/components/IconButton';
+import { Output, OutputType, ShowSign } from '@/components/Output';
+import { TableCell } from '@/components/Table/TableCell';
+
+import { MustBigNumber } from '@/lib/numbers';
+
+type PositionsMarginCellProps = {
+  isDisabled?: boolean;
+  notionalTotal: SubaccountPosition['notionalTotal'];
+  adjustedMmf: SubaccountPosition['adjustedMmf'];
+};
+
+export const PositionsMarginCell = ({
+  adjustedMmf,
+  isDisabled,
+  notionalTotal,
+}: PositionsMarginCellProps) => {
+  const notionalTotalBN = MustBigNumber(notionalTotal?.current);
+  const adjustedMmfBN = MustBigNumber(adjustedMmf?.current);
+  const margin = notionalTotalBN.times(adjustedMmfBN).toFixed(USD_DECIMALS);
```

Comment:
is the `.toFixed(USD_DECIMALS)` needed if we're already using the `Fiat` output type for `Output`?